doc/setup-commands.rst

```diff
@@ -1399,3 +1399,63 @@ This command takes the following option:
 
 
 .. include:: references.inc
+
+runhaskell Setup.hs repl 
+------------------------
+
+Open an interpreter session for the given component.
+
+Usage: 
+
+.. program:: runhaskell Setup.hs repl [COMPONENT] [FLAGS]
+
+If the current directory contains no package, ignores COMPONENT parameters and
+opens an interactive interpreter session; if a sandbox is present, its package
```

Comment:
Weren't sandboxes removed?

Reply:
Sorry, I should have checked the latest document. I just checked document for stable version, and sandboxes were removed indeed. I wrote the document simply based on the output of `runhaskell Setup.hs repl --help`. So, I guess the `--help` for Setup.hs also need to be refined.

Reply:
good catch both of you! 

@Sdywolf, if you could update `--help` in the same PR, that'd be awesome!

Reply:
thx, I'll try to update it in a new PR.